pkgs/tools/audio/beets/plugins/alternatives.nix

```diff
@@ -43,5 +43,8 @@ python3Packages.buildPythonApplication rec {
       lovesegfault
     ];
     license = lib.licenses.mit;
+    # Tests fail on darwin with:
+    # Exception: Unsupported system: Darwin
+    platforms = lib.platforms.linux;
```

Comment:
In version 0.13.1 this shouldn't fail. See:

https://github.com/geigerzaehler/beets-alternatives/compare/v0.13.0..v0.13.1#diff-4dd682d5c858fe7d3f39400e3da7a0ea9e77232bab93c6d51e5cb523844665b2L248